lib/screens/view_selector.dart

```diff
@@ -56,17 +64,27 @@ class _ViewSelectorState extends State<ViewSelector> {
                 },
               );
             }
-
+            print(_views);
             if (_views.isEmpty) {
               _views.addEntries(
                 snapshot.data!
                     .where((element) => element.collectionType != "playlists")
-                    .map((e) => MapEntry(e, e.collectionType == "music")),
+                    .map(
+                      (e) => MapEntry(
+                        e,
+                        e.collectionType == "music" &&
+                            (_finampUserHelper.currentUser!.views.values.where(
+                                  (a) => e.id == a.id,
+                                )).length ==
+                                1,
```

Comment:
We should keep having all libraries selected by default on fresh installs.
```suggestion
                        e.collectionType == "music" &&
                            (_finampUserHelper.currentUser!.views.isEmpty||_finampUserHelper.currentUser!.views.keys.contains(e.id)),
```